gamemodes/core/commands/cmds_cuff.inc

```diff
@@ -0,0 +1,69 @@
+CMD:cuffcw(playerid)
+{
+    new ClosesWar = Player_GetClosestWanted(playerid);
+    new
+        Float:x,
+        Float:y,
+        Float:z;
+    GetPlayerPos(playerid, x, y, z);
+    if(Player_GetClass(playerid) != TEAM_POLICE) {
+        return SendErrorMsg(playerid, "Only cops can use this command!");
+    }
+    if(!IsPlayerConnected(ClosesWar)) {
+        return SendErrorMsg(playerid, "Player is not connected!");
+    }
+    if(!IsPlayerInRangeOfPoint(ClosesWar, 5.0, x, y, z)) {
```

Comment:
``Player_GetClosestWanted`` returns -1 if there are not wanted players nearly.

Better to do

```c
if(ClosestWar == -1)  {
    return SendErrorMsg(playerid, "There's no warrant near you");
}
```

Reply:
Done.